dbt/models/reporting/schema.yml

```diff
@@ -235,6 +235,24 @@ models:
               (year >= '2021' OR (
               year = '2020' AND stage_name = 'BOR CERTIFIED')
               )
+      - accepted_range:
+          name: reporting_vw_pin_value_long_exempt_parcels_values
+          column_name: tot
+          config:
+            tags:
+              - test_qc_iasworld
+          max_value: 1
+          where: |
+            class = 'EX' AND CAST(year AS int) = {{ var('test_qc_year_start') }}
+          additional_select_columns:
+            - {column: pin, alias: parid}
+            - class
+            - stage_name
+            - {column: year, alias: taxyr}
+          meta:
+            table_name: asmt_all
+            description: exempt parcels should not have AVs greater than 1
+            TEST_CATEGORIES: test_accepted_range
```

Comment:
**[Nitpick, non-blocking]** I don't think we need to override the category in this case, since `accepted_range` already has a good default in `TEST_CATEGORIES`:

https://github.com/ccao-data/data-architecture/blob/4b1b86885058fc2fae5e623afe82b2f6ed856cf4/.github/scripts/transform_dbt_test_results.py#L56

But if we do decide we need to override the category, we should use the `category` attribute instead of `TEST_CATEGORIES` -- `TEST_CATEGORIES` is the name of the default variable dict in the script, but it reads from `meta.category` for overrides:

https://github.com/ccao-data/data-architecture/blob/4b1b86885058fc2fae5e623afe82b2f6ed856cf4/.github/scripts/transform_dbt_test_results.py#L1252-L1256

```suggestion
            category: test_accepted_range
```